python/README.md

```diff
@@ -1,27 +1,30 @@
-# Foxglove Studio WebSocket server
+# Foxglove WebSocket protocol in Python
 
-This package provides a server implementation of the [Foxglove Studio WebSocket protocol](https://github.com/foxglove/ws-protocol), enabling [Foxglove Studio](https://github.com/foxglove/studio) to ingest arbitrary “live” streamed data.
-
-The protocol is encoding-agnostic, i.e. it can support Protobuf messages, JSON messages, etc. (as long as the desired encoding is supported by both client and server).
+The Foxglove WebSocket protocol is encoding-agnostic, as long as the desired encoding is supported by both client and server. It currently supports Protobuf and JSON messages.
 
 ## Installation
 
+Install the `foxglove_websocket` Python package:
+
 ```
-$ pip install foxglove-websocket
+$ pip install foxglove_websocket
```

Comment:
I was surprised this works... I guess it is installable via both names, `foxglove-websocket` and  `foxglove_websocket`. I believe it is conventional for package names to use dashes, and apparently PyPI will [actually convert underscores to dashes](https://www.python.org/dev/peps/pep-0503/#normalized-names). PEP 8 also says modules (the thing you import) use underscores, while underscores in package names are discouraged: https://www.python.org/dev/peps/pep-0008/#package-and-module-names  So I think we should keep it with a dash here.

https://groups.google.com/g/comp.lang.python/c/Y5zcSR7wn7c
https://gist.github.com/gene1wood/9472a9d0dffce1a56d6e796afc6539b8
https://github.com/ros/rosdistro/issues/18116